aws_ecr.tf

```diff
@@ -0,0 +1,52 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+resource "aws_iam_user" "user" {
+  name = "${var.team_name}-ecr-system-account"
+  path = "/teams/${var.team_name}/"
+}
+
+resource "aws_iam_access_key" "key" {
+  user = "${aws_iam_user.user.name}"
+}
+
+resource "aws_iam_policy" "policy" {
+  name        = "${var.team_name}-ecr-read-write"
+  path        = "/teams/${var.team_name}/"
+  description = "ECR policy for team ${var.team_name}"
+
+  policy = <<EOF
```

Comment:
I think the policy would be a bit more readable if written as an [`aws_iam_policy_document`](https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html).

Reply:
this was changed to use  aws_iam_policy_document now